test/IndexTestHelpers.h

```diff
@@ -70,7 +70,8 @@ inline Index makeTestIndex(
     std::optional<std::string> turtleInput = std::nullopt,
     bool loadAllPermutations = true, bool usePatterns = true,
     bool usePrefixCompression = true,
-    ad_utility::MemorySize blocksizePermutations = 16_B) {
+    ad_utility::MemorySize blocksizePermutations = 16_B,
+    bool createTextIndex = false) {
```

Comment:
You can even set the default to `true`. No harm in building those indices.

Reply:
No unfortunately not. If set to true many other tests that use this helper function crash. That's why I added the bool in the first place.